renderdoc/driver/vulkan/wrappers/vk_dynamic_funcs.cpp

```diff
@@ -24,6 +24,14 @@
 
 #include "../vk_core.h"
 
+#define ILLEGAL_EDS3_CALL(func)                                            \
+  RDCERR("Aborting the application due to illegal function call: " #func); \
+  abort();
+
+#define ILLEGAL_EDS3_SER(func)                                                 \
+  RDCERR("Calling " #func " is illegal and should not have been serialised."); \
+  return false;
```

Comment:
This macro can be removed once the non-supported functions are otherwise removed as with the other comments.